README.md

```diff
@@ -68,7 +64,7 @@ The adapter is turned on or off via the native SU2 configuration file. If it is
 
 If multiple interfaces exist, the names of all related entries (`PRECICE_WETSURFACE_MARKER_NAME`, `PRECICE_READ_DATA_NAME`,`PRECICE_WRITE_DATA_NAME`, `PRECICE_MESH_NAME`) must be appended by consecutive numbers. Hence, the names (also in the geometry file) need to be alike differing only by the appending number, which must be successively increasing from zero. E.g. for three interfaces, the marker name could be defined as `PRECICE_WETSURFACE_MARKER_NAME= wetSurface` in the SU2 configuration file, while the markers in the geometry file would need to be named *wetSurface*, *wetSurface1* and *wetSurface2*.
 
-Moreover, in the SU2 configuration file grid movement must be allowed: `GRID_MOVEMENT= YES` and the type of grid movement must be set correctly for preCICE-coupled simulations: `GRID_MOVEMENT_KIND= PRECICE_MOVEMENT`. Also, the boundary, which is allowed to move needs to be specified. Here the name of the FSI-interface marker including its appending identifying number as stated above needs to be used, e.g., `MARKER_MOVING= ( wetSurface0 )`. If multiple FSI-interfaces exist (as in the example above), this may look like `MARKER_MOVING= ( wetSurface, wetSurface1, wetSurface2 )`.
+Moreover, in the SU2 configuration file grid movement the type of grid movement must be set correctly for preCICE-coupled simulations: `SURFACE_MOVEMENT= PRECICE_MOVEMENT`. Also, the boundary, which is allowed to move needs to be specified. Here the name of the FSI-interface marker including its appending identifying number as stated above needs to be used, e.g., `MARKER_MOVING= ( wetSurface0 )`. If multiple FSI-interfaces exist (as in the example above), this may look like `MARKER_MOVING= ( wetSurface, wetSurface1, wetSurface2 )`.
 
 ### Running the Adapted SU2 Executable
```

Comment:
I cannot run `SU2_CFD` on my Ubuntu 22.04 (I can run the previous version, though):

```bash
~/repos/precice/tutorials/perpendicular-flap/fluid-su2 [develop]$ ./run.sh 
[tickly:159675] *** An error occurred in MPI_Win_create
[tickly:159675] *** reported by process [2347040769,0]
[tickly:159675] *** on communicator MPI_COMM_WORLD
[tickly:159675] *** MPI_ERR_WIN: invalid window
[tickly:159675] *** MPI_ERRORS_ARE_FATAL (processes in this communicator will now abort,
[tickly:159675] ***    and potentially your MPI job)
~/repos/precice/tutorials/perpendicular-flap/fluid-su2 [develop]$ which SU2_CFD
/usr/local/bin/SU2_CFD
~/repos/precice/tutorials/perpendicular-flap/fluid-su2 [develop]$ ldd /usr/local/bin/SU2_CFD
	linux-vdso.so.1 (0x00007fff73fa1000)
	libz.so.1 => /lib/x86_64-linux-gnu/libz.so.1 (0x00007f37be7b8000)
	libmpi.so.40 => /lib/x86_64-linux-gnu/libmpi.so.40 (0x00007f37be681000)
	libmpi_cxx.so.40 => /lib/x86_64-linux-gnu/libmpi_cxx.so.40 (0x00007f37be667000)
	libprecice.so.2 => /home/gc/repos/precice/precice/build/libprecice.so.2 (0x00007f37be2aa000)
	libstdc++.so.6 => /lib/x86_64-linux-gnu/libstdc++.so.6 (0x00007f37be07e000)
	libm.so.6 => /lib/x86_64-linux-gnu/libm.so.6 (0x00007f37bdf97000)
	libgcc_s.so.1 => /lib/x86_64-linux-gnu/libgcc_s.so.1 (0x00007f37bdf77000)
	libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007f37bdd4f000)
	libopen-rte.so.40 => /lib/x86_64-linux-gnu/libopen-rte.so.40 (0x00007f37bdc92000)
	libopen-pal.so.40 => /lib/x86_64-linux-gnu/libopen-pal.so.40 (0x00007f37bdbdf000)
	libhwloc.so.15 => /lib/x86_64-linux-gnu/libhwloc.so.15 (0x00007f37bdb81000)
	libboost_log_setup.so.1.74.0 => /lib/x86_64-linux-gnu/libboost_log_setup.so.1.74.0 (0x00007f37bdae3000)
	libboost_program_options.so.1.74.0 => /lib/x86_64-linux-gnu/libboost_program_options.so.1.74.0 (0x00007f37bda9e000)
	libxml2.so.2 => /lib/x86_64-linux-gnu/libxml2.so.2 (0x00007f37bd8bc000)
	libpython3.10.so.1.0 => /lib/x86_64-linux-gnu/libpython3.10.so.1.0 (0x00007f37bd2e2000)
	libboost_log.so.1.74.0 => /lib/x86_64-linux-gnu/libboost_log.so.1.74.0 (0x00007f37bd234000)
	libboost_filesystem.so.1.74.0 => /lib/x86_64-linux-gnu/libboost_filesystem.so.1.74.0 (0x00007f37bd214000)
	libboost_thread.so.1.74.0 => /lib/x86_64-linux-gnu/libboost_thread.so.1.74.0 (0x00007f37bd1f2000)
	libpetsc_real.so.3.15 => /usr/lib/petscdir/petsc3.15/x86_64-linux-gnu-real/lib/libpetsc_real.so.3.15 (0x00007f37bbe74000)
	/lib64/ld-linux-x86-64.so.2 (0x00007f37bf998000)
	libevent_core-2.1.so.7 => /lib/x86_64-linux-gnu/libevent_core-2.1.so.7 (0x00007f37bbe3f000)
	libevent_pthreads-2.1.so.7 => /lib/x86_64-linux-gnu/libevent_pthreads-2.1.so.7 (0x00007f37bbe38000)
	libudev.so.1 => /lib/x86_64-linux-gnu/libudev.so.1 (0x00007f37bbe0e000)
	libboost_regex.so.1.74.0 => /lib/x86_64-linux-gnu/libboost_regex.so.1.74.0 (0x00007f37bbd1b000)
	libicuuc.so.70 => /lib/x86_64-linux-gnu/libicuuc.so.70 (0x00007f37bbb20000)
	liblzma.so.5 => /lib/x86_64-linux-gnu/liblzma.so.5 (0x00007f37bbaf5000)
	libexpat.so.1 => /lib/x86_64-linux-gnu/libexpat.so.1 (0x00007f37bbac2000)
	libHYPRE-2.22.1.so => /lib/x86_64-linux-gnu/libHYPRE-2.22.1.so (0x00007f37bb6aa000)
	libdmumps-5.4.so => /lib/x86_64-linux-gnu/libdmumps-5.4.so (0x00007f37bb496000)
	libscalapack-openmpi.so.2.1 => /lib/x86_64-linux-gnu/libscalapack-openmpi.so.2.1 (0x00007f37baeee000)
	libumfpack.so.5 => /lib/x86_64-linux-gnu/libumfpack.so.5 (0x00007f37bae42000)
	libamd.so.2 => /lib/x86_64-linux-gnu/libamd.so.2 (0x00007f37bae35000)
	libcholmod.so.3 => /lib/x86_64-linux-gnu/libcholmod.so.3 (0x00007f37bad54000)
	libklu.so.1 => /lib/x86_64-linux-gnu/libklu.so.1 (0x00007f37bad25000)
	libsuperlu.so.5 => /lib/x86_64-linux-gnu/libsuperlu.so.5 (0x00007f37bacb3000)
	libsuperlu_dist.so.7 => /lib/x86_64-linux-gnu/libsuperlu_dist.so.7 (0x00007f37bab4f000)
	libtrilinos_ml.so.13.2 => /lib/x86_64-linux-gnu/libtrilinos_ml.so.13.2 (0x00007f37ba896000)
	libfftw3.so.3 => /lib/x86_64-linux-gnu/libfftw3.so.3 (0x00007f37ba679000)
	libfftw3_mpi.so.3 => /lib/x86_64-linux-gnu/libfftw3_mpi.so.3 (0x00007f37ba65f000)
	liblapack.so.3 => /lib/x86_64-linux-gnu/liblapack.so.3 (0x00007f37b9f26000)
	libblas.so.3 => /lib/x86_64-linux-gnu/libblas.so.3 (0x00007f37b9e80000)
	libptscotch-6.1.so => /lib/x86_64-linux-gnu/libptscotch-6.1.so (0x00007f37b9dad000)
	libhdf5_openmpi.so.103 => /lib/x86_64-linux-gnu/libhdf5_openmpi.so.103 (0x00007f37b99f8000)
	libOpenCL.so.1 => /lib/x86_64-linux-gnu/libOpenCL.so.1 (0x00007f37b99e4000)
	libyaml-0.so.2 => /lib/x86_64-linux-gnu/libyaml-0.so.2 (0x00007f37b99c3000)
	libX11.so.6 => /lib/x86_64-linux-gnu/libX11.so.6 (0x00007f37b9883000)
	libmpi_mpifh.so.40 => /lib/x86_64-linux-gnu/libmpi_mpifh.so.40 (0x00007f37b981d000)
	libgfortran.so.5 => /lib/x86_64-linux-gnu/libgfortran.so.5 (0x00007f37b9542000)
	libicui18n.so.70 => /lib/x86_64-linux-gnu/libicui18n.so.70 (0x00007f37b9211000)
	libicudata.so.70 => /lib/x86_64-linux-gnu/libicudata.so.70 (0x00007f37b75f3000)
	libmumps_common-5.4.so => /lib/x86_64-linux-gnu/libmumps_common-5.4.so (0x00007f37b7597000)
	libsuitesparseconfig.so.5 => /lib/x86_64-linux-gnu/libsuitesparseconfig.so.5 (0x00007f37b7592000)
	libcolamd.so.2 => /lib/x86_64-linux-gnu/libcolamd.so.2 (0x00007f37b7589000)
	libccolamd.so.2 => /lib/x86_64-linux-gnu/libccolamd.so.2 (0x00007f37b757a000)
	libcamd.so.2 => /lib/x86_64-linux-gnu/libcamd.so.2 (0x00007f37b756e000)
	libmetis.so.5 => /lib/x86_64-linux-gnu/libmetis.so.5 (0x00007f37b7504000)
	libgomp.so.1 => /lib/x86_64-linux-gnu/libgomp.so.1 (0x00007f37b74ba000)
	libbtf.so.1 => /lib/x86_64-linux-gnu/libbtf.so.1 (0x00007f37b74b4000)
	libptscotchparmetis-6.1.so => /lib/x86_64-linux-gnu/libptscotchparmetis-6.1.so (0x00007f37b74ab000)
	libCombBLAS.so.1.16.0 => /lib/x86_64-linux-gnu/libCombBLAS.so.1.16.0 (0x00007f37b7492000)
	libtrilinos_ifpack.so.13.2 => /lib/x86_64-linux-gnu/libtrilinos_ifpack.so.13.2 (0x00007f37b7300000)
	libtrilinos_amesos.so.13.2 => /lib/x86_64-linux-gnu/libtrilinos_amesos.so.13.2 (0x00007f37b72ab000)
	libtrilinos_galeri-epetra.so.13.2 => /lib/x86_64-linux-gnu/libtrilinos_galeri-epetra.so.13.2 (0x00007f37b7265000)
	libtrilinos_aztecoo.so.13.2 => /lib/x86_64-linux-gnu/libtrilinos_aztecoo.so.13.2 (0x00007f37b71df000)
	libtrilinos_zoltan.so.13.2 => /lib/x86_64-linux-gnu/libtrilinos_zoltan.so.13.2 (0x00007f37b70fd000)
	libtrilinos_epetraext.so.13.2 => /lib/x86_64-linux-gnu/libtrilinos_epetraext.so.13.2 (0x00007f37b6f95000)
	libscotch-6.1.so => /lib/x86_64-linux-gnu/libscotch-6.1.so (0x00007f37b6f06000)
	libtrilinos_epetra.so.13.2 => /lib/x86_64-linux-gnu/libtrilinos_epetra.so.13.2 (0x00007f37b6da3000)
	libtrilinos_teuchosparameterlist.so.13.2 => /lib/x86_64-linux-gnu/libtrilinos_teuchosparameterlist.so.13.2 (0x00007f37b6a82000)
	libtrilinos_teuchoscore.so.13.2 => /lib/x86_64-linux-gnu/libtrilinos_teuchoscore.so.13.2 (0x00007f37b6a25000)
	libptscotcherr-6.1.so => /lib/x86_64-linux-gnu/libptscotcherr-6.1.so (0x00007f37b6a20000)
	libbz2.so.1.0 => /lib/x86_64-linux-gnu/libbz2.so.1.0 (0x00007f37b6a0d000)
	libcrypto.so.3 => /lib/x86_64-linux-gnu/libcrypto.so.3 (0x00007f37b65cb000)
	libcurl.so.4 => /lib/x86_64-linux-gnu/libcurl.so.4 (0x00007f37b6525000)
	libsz.so.2 => /lib/x86_64-linux-gnu/libsz.so.2 (0x00007f37b651e000)
	libxcb.so.1 => /lib/x86_64-linux-gnu/libxcb.so.1 (0x00007f37b64f4000)
	libquadmath.so.0 => /lib/x86_64-linux-gnu/libquadmath.so.0 (0x00007f37b64ac000)
	libpord-5.4.so => /lib/x86_64-linux-gnu/libpord-5.4.so (0x00007f37b6493000)
	libtrilinos_trilinosss.so.13.2 => /lib/x86_64-linux-gnu/libtrilinos_trilinosss.so.13.2 (0x00007f37b6463000)
	libtrilinos_teuchosremainder.so.13.2 => /lib/x86_64-linux-gnu/libtrilinos_teuchosremainder.so.13.2 (0x00007f37b6459000)
	libtrilinos_teuchosnumerics.so.13.2 => /lib/x86_64-linux-gnu/libtrilinos_teuchosnumerics.so.13.2 (0x00007f37b643c000)
	libtrilinos_teuchoscomm.so.13.2 => /lib/x86_64-linux-gnu/libtrilinos_teuchoscomm.so.13.2 (0x00007f37b6365000)
	libtrilinos_triutils.so.13.2 => /lib/x86_64-linux-gnu/libtrilinos_triutils.so.13.2 (0x00007f37b6307000)
	libscotcherr-6.1.so => /lib/x86_64-linux-gnu/libscotcherr-6.1.so (0x00007f37b6302000)
	libtrilinos_teuchosparser.so.13.2 => /lib/x86_64-linux-gnu/libtrilinos_teuchosparser.so.13.2 (0x00007f37b62a0000)
	libtrilinos_kokkoscore.so.13.2 => /lib/x86_64-linux-gnu/libtrilinos_kokkoscore.so.13.2 (0x00007f37b6216000)
	libnghttp2.so.14 => /lib/x86_64-linux-gnu/libnghttp2.so.14 (0x00007f37b61ec000)
	libidn2.so.0 => /lib/x86_64-linux-gnu/libidn2.so.0 (0x00007f37b61cb000)
	librtmp.so.1 => /lib/x86_64-linux-gnu/librtmp.so.1 (0x00007f37b61ac000)
	libssh.so.4 => /lib/x86_64-linux-gnu/libssh.so.4 (0x00007f37b613d000)
	libpsl.so.5 => /lib/x86_64-linux-gnu/libpsl.so.5 (0x00007f37b6129000)
	libssl.so.3 => /lib/x86_64-linux-gnu/libssl.so.3 (0x00007f37b6085000)
	libgssapi_krb5.so.2 => /lib/x86_64-linux-gnu/libgssapi_krb5.so.2 (0x00007f37b6031000)
	libldap-2.5.so.0 => /lib/x86_64-linux-gnu/libldap-2.5.so.0 (0x00007f37b5fd2000)
	liblber-2.5.so.0 => /lib/x86_64-linux-gnu/liblber-2.5.so.0 (0x00007f37b5fc1000)
	libzstd.so.1 => /lib/x86_64-linux-gnu/libzstd.so.1 (0x00007f37b5ef0000)
	libbrotlidec.so.1 => /lib/x86_64-linux-gnu/libbrotlidec.so.1 (0x00007f37b5ee2000)
	libaec.so.0 => /lib/x86_64-linux-gnu/libaec.so.0 (0x00007f37b5ed9000)
	libXau.so.6 => /lib/x86_64-linux-gnu/libXau.so.6 (0x00007f37b5ed3000)
	libXdmcp.so.6 => /lib/x86_64-linux-gnu/libXdmcp.so.6 (0x00007f37b5ecb000)
	libunistring.so.2 => /lib/x86_64-linux-gnu/libunistring.so.2 (0x00007f37b5d1f000)
	libgnutls.so.30 => /lib/x86_64-linux-gnu/libgnutls.so.30 (0x00007f37b5b34000)
	libhogweed.so.6 => /lib/x86_64-linux-gnu/libhogweed.so.6 (0x00007f37b5aec000)
	libnettle.so.8 => /lib/x86_64-linux-gnu/libnettle.so.8 (0x00007f37b5aa6000)
	libgmp.so.10 => /lib/x86_64-linux-gnu/libgmp.so.10 (0x00007f37b5a24000)
	libkrb5.so.3 => /lib/x86_64-linux-gnu/libkrb5.so.3 (0x00007f37b5957000)
	libk5crypto.so.3 => /lib/x86_64-linux-gnu/libk5crypto.so.3 (0x00007f37b5928000)
	libcom_err.so.2 => /lib/x86_64-linux-gnu/libcom_err.so.2 (0x00007f37b5922000)
	libkrb5support.so.0 => /lib/x86_64-linux-gnu/libkrb5support.so.0 (0x00007f37b5914000)
	libsasl2.so.2 => /lib/x86_64-linux-gnu/libsasl2.so.2 (0x00007f37b58f9000)
	libbrotlicommon.so.1 => /lib/x86_64-linux-gnu/libbrotlicommon.so.1 (0x00007f37b58d6000)
	libbsd.so.0 => /lib/x86_64-linux-gnu/libbsd.so.0 (0x00007f37b58bc000)
	libp11-kit.so.0 => /lib/x86_64-linux-gnu/libp11-kit.so.0 (0x00007f37b5781000)
	libtasn1.so.6 => /lib/x86_64-linux-gnu/libtasn1.so.6 (0x00007f37b5769000)
	libkeyutils.so.1 => /lib/x86_64-linux-gnu/libkeyutils.so.1 (0x00007f37b5762000)
	libresolv.so.2 => /lib/x86_64-linux-gnu/libresolv.so.2 (0x00007f37b574e000)
	libmd.so.0 => /lib/x86_64-linux-gnu/libmd.so.0 (0x00007f37b573f000)
	libffi.so.8 => /lib/x86_64-linux-gnu/libffi.so.8 (0x00007f37b5732000)
```

Any clue what I could be missing? This is not specific to the tutorial, I cannot start it in general.

Reply:
Two things:

1. One thing to watch out for that SU2 does not successfully implement at least Intel MPI (https://www.cfd-online.com/Forums/su2-installation/228428-su2-parallel-install-build-source-failed.html#post776838) with default meson settings as you show above. I have had to run with the additional settings below to get it to work:

```
export CC=$(which mpicc)
export CXX=$(which mpicxx)
export MPICC=$(which mpicc)
export MPICXX=$(which mpicxx)

./meson.py build -Dcustom-mpi=true --prefix=...
./ninja -C build install
```

I am not sure how it does with other MPIs, but something similar may be needed. I've tried numerous different things that didn't involve needing to set `-Dcustom-mpi` but have never been successful.

I would try this above and see if it helps?

2. This is exactly the perpendicular flap tutorial case but a lot of names of options in the config file have changed since last SU2. The `tests` folder in this branch includes an updated config file that works for new SU2.